### ISSUE TITLE
Adding a wrap() method that returns a list of lines

### ISSUE THREAD

**Issue by vandeseer** [davidmoten/word-wrap#6]
Hi!

First of all: Very nice library! I am thinking of integrating it into a project of my own – but in order to so I would need to have a method that allows me to get the single lines as a list when I `wrap()` my strings. 

Are there any plans to add such a feature? Could it be added easily? 

Thanks!
Stefan

**Comment by davidmoten** [comment]
Glad you like it. Sure I can add a convenience method later today. In the meantime just use:

```java
String wrappedText = ...
List<String> list = Arrays.asList(wrappedText.split("\n"));
```

**Comment by davidmoten** [comment]
I've just released v 0.1.9 to Maven Central. It includes methods `wrapToList()` and `wrap(LineConsumer)`. That should cover your request.

**Comment by vandeseer** [comment]
Cool! Thanks a lot! 👍 😃
